src/parser/jobs/drk/index.js

```diff
@@ -24,5 +24,10 @@ export default new Meta({
 			Changes: () => <>Initial release of 5.0 support for Dark Knight</>,
 			contributors: [CONTRIBUTORS.AZARIAH],
 		},
+		{
```

Comment:
only just noticed (muh bad), but while it doesn't _matter_ per se (cus changelog is sorted), generally add changelogs to the top.